sdk/textanalytics/Azure.AI.TextAnalytics/src/RecognizeCustomEntitiesAction.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary>
+    /// Configurations that allow callers to specify details about how to execute
+    /// a Recognize Custom Entities action in a set of documents.
+    /// Project Name and Deployment Name are required.
```

Comment:
Consider removing this sentence as right now you are just describing what the class is about, not the behavior of ir